### PR TITLE
add test_histogram_ci_plot

### PR DESCRIPTION
**Pull request by zackt113**
As per the chat from Slack, I will raise the PR first and see how it goes! Thanks

### CODE REVIEW

File: tests/test_data_vis.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    --------
+    None
+        The test should pass and no asserts should be displayed.
+        
+    6 tests in total
+    """
+    
+    # tests with invalid input type of title
+    with raises(TypeError) as e:
+        histogram_ci_plot([1, 2, 3, 4, 5, 6, 7], 1000 ,n=100, ci_level=0.95, title = 123)
```
Comment (gauthampughazhendhi) [inline tests/test_data_vis.py]:
```suggestion
        histogram_ci_plot([1, 2, 3, 4, 5, 6, 7], 1000, n=100, ci_level=0.95, title=123)
```

Comment (gauthampughazhendhi) [reply tests/test_data_vis.py]: @zackt113 can you add this small formatting change everywhere?

Comment (zackt113) [reply tests/test_data_vis.py]: Fixed it! Thanks @gauthampughazhendhi

**Review by gauthampughazhendhi — APPROVED** [review]

**Comment by gauthampughazhendhi** [comment]
Looks good to me.

**Comment by voremargot** [comment]
things look good to me too! I am going to merge them!

**Pull request merged**